mex/common/backend_api/connector.py

```diff
@@ -109,14 +143,63 @@ def fetch_merged_items(
             endpoint="merged-item",
             params={
                 "q": query_string,
+                "identifier": identifier,
                 "entityType": entity_type,
-                "hadPrimarySource": had_primary_source,
+                "referencedIdentifier": referenced_identifier,
+                "referenceField": reference_field,
                 "skip": str(skip),
                 "limit": str(limit),
             },
         )
         return PaginatedItemsContainer[AnyMergedModel].model_validate(response)
 
+    def fetch_all_merged_items(
+        self,
+        *,
+        query_string: str | None = None,
+        identifier: str | None = None,
+        entity_type: list[str] | None = None,
+        referenced_identifier: list[str] | None = None,
+        reference_field: str | None = None,
+    ) -> Generator[AnyMergedModel, None, None]:
+        """Fetch all merged items that match the given set of filters.
+
+        Args:
+            query_string: Full-text search query
+            identifier: Optional merged item identifier filter
+            entity_type: The items' entityType
+            referenced_identifier: Optional merged item identifiers filter
+            reference_field: Optional field name to filter for
```

Comment:
All arguments are optional. Can we either add "optional" to all or remove it for all args and assume that devs infer that args with a default are optional.

Reply:
fixed dc9451d